staging-service/src/main/java/com/github/cafdataprocessing/services/staging/StagingApi.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2019-2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.github.cafdataprocessing.services.staging;
+
+import com.github.cafdataprocessing.services.staging.models.BatchList;
+import com.github.cafdataprocessing.services.staging.models.StatusResponse;
+import io.swagger.annotations.*;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import javax.validation.Valid;
+import javax.validation.constraints.*;
+import org.springframework.web.multipart.MultipartHttpServletRequest;
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2022-05-10T15:07:03.708+01:00[Europe/London]")
+@Api(value = "Staging", description = "the Staging API")
+public interface StagingApi {
+
+    @ApiOperation(value = "Upload documents. The batch will be automatically created if it doesn't already exist.", nickname = "createOrReplaceBatch", notes = "", tags={  })
+    @ApiResponses(value = { 
+        @ApiResponse(code = 200, message = "Successfully uploaded batch of documents."),
+        @ApiResponse(code = 400, message = "The request could not be processed because one or more arguments are invalid."),
+        @ApiResponse(code = 500, message = "The request failed due to an unexpected server error.") })
+    @RequestMapping(value = "/batches/{batchId}",
+        consumes = { "multipart/mixed" },
+        method = RequestMethod.PUT)
+    ResponseEntity<Void> createOrReplaceBatch(@ApiParam(value = "Identifies the tenant making the request." ,required=true) @RequestHeader(value="X-TENANT-ID", required=true) String X_TENANT_ID,@Size(min=1) @ApiParam(value = "Identifies the batch.",required=true) @PathVariable("batchId") String batchId,@ApiParam(value = ""  ) MultipartHttpServletRequest request);
```

Comment:
Copied and modified in order to change `@Valid @RequestBody Object body` to `MultipartHttpServletRequest request`

Reply:
Actually, I remember there's a way to do this in maven using the `gmavenplus-plugin`, I'll have a look at this on Monday